pyanaconda/dbus/__init__.py

```diff
@@ -0,0 +1,16 @@
+import os
+import pydbus
+
+def get_bus():
+    """Get a DBUS bus connection corresponding to the current environment.
+
+    During normal usage this function should return connection to the system bus,
+    but during testing/development a custom bus might be used.
+    So just always connect to the bus specified by the DBUS_STARTER_ADDRESS
+    environmental variable.
+    """
+    bus_address = os.environ.get("DBUS_STARTER_ADDRESS")
+    if bus_address:
+        return pydbus.connect(os.environ["DBUS_STARTER_ADDRESS"])
+    else:
+        return pydbus.SystemBus()
```

Comment:
Yeah, it could mean a serious anomaly - let's log an error & raise an exception.
We can always change it if a valid scenario where "DBUS_STARTER_ADDRESS" is not set shows up.